zjit/src/options.rs

```diff
@@ -114,7 +114,7 @@ pub const ZJIT_OPTIONS: &[(&str, &str)] = &[
     ("--zjit-call-threshold=num",
                      "Number of calls to trigger JIT (default: 2)."),
     ("--zjit-num-profiles=num",
-                     "Number of profiled calls before JIT (default: 1, max: 255)."),
+                     "Number of profiled calls before JIT (default: 1, max: 2^32-1)."),
```

Comment:
The actual number is `4,294,967,295`, but I feel it's not particularly helpful to state it

Reply:
Can we just drop `, max: 2^32-1` from the description? Unlike 2^16, you effectively never hit it.